src/CameraControls/useCenterGraph.ts

```diff
@@ -41,6 +41,8 @@ export const useCenterGraph = ({
   disabled
 }: CenterGraphInput): CenterGraphOutput => {
   const nodes = useStore(state => state.nodes);
+  const isCentered = useStore(state => state.isCentered);
```

Comment:
You don't need to put this in the store based on the current useage, u could just return in state.